### ISSUE TITLE
Better Volume - VSA индикатор объемов

### ISSUE THREAD

**Issue by targowes** [SBProX/SBProX-Dev#1888]
Добрый день. Это очень простой индикатор, основанный на размере свечи и объеме в этой свече. Он уже давно есть для MT4. Вчера он появился в АТАСе. 
Описание: 
Индикатор Better Volume использует текущий тиковый объем вместе с диапазоном свечи (спрэдом), сравнивая их с предыдущими значениями (в настройках по умолчанию это 20 последних свечей), выдавая следующие сигналы:

● красного цвета. Большой объём + бычья свеча, у которой большой спрэд. Сигнал появляется при зарождении и окончании восходящего тренда, также во время коррекции (откатов) на медвежьем рынке.

● белого цвета. Большой объём + медвежья свеча, у которой большой спрэд. Сигнал появляется при зарождении и окончании нисходящего тренда, также во время коррекции (откатов) на бычьем рынке.

● зелёного цвета. Большой объём + свеча с небольшим спрэдом. Как правило, возникает при окончании трендового движения, когда маркетмейкер “сбрасывает” свои сделки обычным трейдерам, которые понесут убытки, поскольку игроки открывают свои позиции на окончании тренда, как раз перед тем, как цена развернётся, а также в момент фиксации прибыли на коррекции (в середине трендового движения).

● пурпурного цвета. Группа свечей с большим объемом. Свечи имеют большой и малый спрэд. Данный сигнал появляется редко, обычно это происходит на коррекции (откате).

● жёлтого цвета. Свечи, имеющие малый тиковый объем. Обычное явление для окончания тренда и во время коррекции.

Когда сигналов нет, столбики гистограммы окрашены в голубой цвет.

Более подробно можно посмотреть на вебинаре для атаса:
https://www.youtube.com/watch?v=lrMWNliNoVk&feature=youtu.be 
начиная с 49:51

Вещь практически незаменимая. Изумительная точность сигналов, при этом очень маленькие стопы.
Сделайте пожалуйста!

**Comment by WebSurfer001** [comment]
Поддерживаю

**Comment by Priuma** [comment]
Тогда уж делать как в Xtick конструктор: бар может выделяться и подать сигнал при условиях: объем, дельта, открытый интерес и т.д., выбираешь условия сам - если, или, и... и т.д. Соответственно,  под каждый инструмент и таймфрейм можно задавать свои условия.
![ооо](https://user-images.githubusercontent.com/78469010/109431346-2cd62400-7a17-11eb-8028-dde8e01f715c.JPG)

**Comment by targowes** [comment]
Да, это было бы не плохо. Хотя, если такое реализовывать, нужно не бар подсвечивать, а значок над ним вешать. Что бы можно было видеть, что допустим на этом баре и дельта сработала, и объем одновременно. Иначе будет не понятно, что именно бар подсветило. Ну и идеально алерт еще повесить на такой сигнал не забыть.

**Comment by Priuma** [comment]
в Xtick это именно так и работает - значок над/под баром, подсветка, алерт. 
А условия любые - и/или, больше/меньше и т.д,

**Comment by targowes** [comment]
Ну, дай Бог, этот топик заметят. Реализовать это совсем несложно, а нам была бы огромная польза.

**Comment by Priuma** [comment]
Реализация конструктора типа Xpaint - это сразу выход на аудиторию VSA/побарников и т.п.

**Comment by targowes** [comment]
Это да, клиентов у платформы сразу прибавится.
